libmscore/measure.cpp

```diff
@@ -469,30 +469,65 @@ qreal Measure::tick2pos(Fraction tck) const
       }
 
 //---------------------------------------------------------
-//   layoutMeasureNumber
+//   showsMeasureNumberInAutoMode
+///    Wheter the measure will show measure number(s) when MeasureNumberMode is set to AUTO
 //---------------------------------------------------------
 
-void Measure::layoutMeasureNumber()
+bool Measure::showsMeasureNumberInAutoMode()
       {
-      bool smn = false;
+      // Measure numbers should not be shown on irregular measures.
+      if (irregular())
+            return false;
+
+      // Measure numbers should not show on first measure unless specified with Sid::showMeasureNumberOne
+      if (!no())
```

Comment:
`no()` isn't very understandable...

Reply:
`mNum()` might be better, but this is not the fault of this PR though.

Reply:
What's funny is that I actually hesitated upon renaming `no()` to `number()` but thought that it would make everyone mad like it did for PR #5746